clang/lib/DPCT/PatternRewriter.cpp

```diff
@@ -388,6 +393,33 @@ static bool isIdentifiedChar(char Char) {
   return false;
 }
 
+static void
+updateExtentionName(const std::string &Input, size_t Next,
+                    std::unordered_map<std::string, std::string> &Bindings) {
+  if (Input.compare(Next, strlen(".cpp"), ".cpp") == 0) {
```

Comment:
How about using llvm API like path::extension?

Reply:
Input is not  a file path, it is a string buffer which include file name, so path::extension does not work.